samples/msgext-bot-SPUploader/README.md

```diff
@@ -83,38 +83,48 @@ Step 1: Setup bot in Service
-	* Navigate to https//{SharePointDomain}/_layouts/15/appregnew.aspx
+	* Navigate to https//{SharePointDomain}.sharepoint.com/sites/{SiteName}/_layouts/15/appregnew.aspx
+	
+		eg: https//xxxxxxxxx.sharepoint.com/sites/TestSite/_layouts/15/appregnew.aspx
 
 <img src="./Docs/Images/SharepointAppCreation.PNG" alt="Bot Channels" width="1000" height="500">
 
-2. Know your Tenant ID and Resource ID
+2. Grant permissions
+	* New Client app has been created in SP Online site, now its time to decide what permissions this app should have on your site. You can grant Site collection, web or even at list level read or write permissions. [Follow docs](https://docs.microsoft.com/en-us/sharepoint/dev/solution-guidance/security-apponly-azureacs) to add permission.
+
+	* Go to  https//{SharePointDomain}.sharepoint.com/sites/{SiteName}/_layouts/15/appinv.aspx and search with ClientID we generated earlier. The application will fetch all other details based on your ClientID.
+
+<img src="./Docs/Images/GrantPermission.PNG" alt="Bot Channels" width="1000" height="500">
+
+3. Know your Tenant ID and Resource ID
 	* It is very important to know your tenant ID for triggering any kind of service calls. You can get your Tenant ID, Resource Id by following below points:
-	* Navigate to https//{SharePointDomain}/_layouts/15/appprincipals.aspx
+	* Navigate to https//{SharePointDomain}.sharepoint.com/sites/{SiteName}/_layouts/15/appprincipals.aspx
 	* You will see Site Collection App Permissions under site settings.
 	* You can check your any App and get the Tenant Id and Resource Id from App Identifier. The part after "@" is your tenant ID and the part before @ is Resource ID
+	* In the appsetting.json you need to add SharePointResource value which is 00000003-0000-0ff1-ce00-000000000000 and it is constant value. [Follow docs](https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/authorization-code-oauth-flow-for-sharepoint-add-ins)
 
 <img src="./Docs/Images/AppPermissions.PNG" alt="Bot Channels" width="1000" height="500">
 
-3. Grant permissions
-	* New Client app has been created in SP Online site, now its time to decide what permissions this app should have on your site. You can grant Site collection, web or even at list level read or write permissions.
-	* Go to  https//{SharePointDomain}/_layouts/15/appinv.aspx and serach with ClientID we generated earlier. The application will fetch all other details based on your ClientID.
-
-<img src="./Docs/Images/GrantPermission.PNG" alt="Bot Channels" width="1000" height="500">
-
 4. Create document library
-	* Create Two document libraries to upload the files into sharepoint doc library
+	* Create Two document libraries to upload the files into sharepoint doc library.
+	* You need to mention these document libraries in appsetting.json as StagingFolder and ApprovedFolder.
 
+<img src="./Docs/Images/CreateDocumentLibrary.png" alt="Create Document Library" width="1000" height="500">
```

Comment:
This image is not loading in preview, can you fix the image src / name?

Reply:
Resolved